gamemodes/terrortown/gamemode/server/sv_entity.lua

```diff
@@ -2,6 +2,10 @@
 -- @module Entity
 -- @ref https://wiki.garrysmod.com/page/Category:Entity
 
+local safeCollisionGroups = {
+	[COLLISION_GROUP_WEAPON] = true
```

Comment:
if there is just one static entry, we don't need to use a table. Just return below `self:GetCollisionGroup() == COLLISION_GROUP_WEAPON`

Reply:
I know, but there are probably other safe groups that can be added at a later time

Reply:
Then we can change it later into a table.

Reply:
On this, I think leaving it as a table would keep it extensible and is no overhead that would have to be considered impactful, if it can even be noticed, which I suppose it won't.